### PR TITLE
Mark 281 remove offchain cancel orders

### PR DESCRIPTION
**Pull request by lorraineS**

### CODE REVIEW

File: app/adapters/use-eoa-txs.ts [hunk 1]
```diff
@@ -44,21 +42,9 @@ export function useEOATxs(): WalletAdapter {
...
-        r,
-        s,
-        v,
-      },
-    }
-
-    return await comethMarketplaceClient.order.cancelOrder(nonce, body)
+  async function cancelOrder({ nonce, nftSwapSdk }: CancelOrderParams) {
+    const tx = await nftSwapSdk?.cancelOrder(nonce, "ERC721")
+    return await tx?.wait()
```
Comment (raphael-verdier) [inline app/adapters/use-eoa-txs.ts]: This function already exists here: https://github.com/cometh-game/marketplace-ui/blob/4120648566db7aa44d997b08a983efa3ebe3cde3/app/adapters/use-cometh-txs.ts#L23 
If needed you can move it into a more generic file, but you should share the same code.

File: services/orders/cancel-buy-offer.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ export type CancelBuyOfferParams = {
 }
 
 export const useCancelBuyOffer = () => {
-  const signer = useSigner()
+  // const signer = useSigner()
```
Comment (raphael-verdier) [inline services/orders/cancel-buy-offer.ts]: Delete

File: services/orders/cancel-buy-offer.ts [hunk 3]
```diff
@@ -38,7 +38,6 @@ export const useCancelBuyOffer = () => {
 
       return await getWalletTxs()?.cancelOrder({
         nonce,
-        signer,
```
Comment (raphael-verdier) [inline services/orders/cancel-buy-offer.ts:41]: How come you don't need the signer anymore?

Comment (lorraineS) [reply services/orders/cancel-buy-offer.ts:41]: NftswapSdk manages it for us https://docs.swapsdk.xyz/readme/managing-orders#cancelling-orders

**Pull request merged**